testing/data_validation/get-expectation-values-for-qubit-operator.py

```diff
@@ -20,12 +20,19 @@
             == "zapata-v1-expectation_values"
         )
 
+        identity_term = workflowresult[key]["expectation-values"]["expectation_values"][
+            "real"
+        ][0]
+        assert isinstance(float(identity_term), float)
+        assert identity_term > 1.5
+        assert identity_term < 2.5
+
         for expectation_value in workflowresult[key]["expectation-values"][
             "expectation_values"
-        ]["real"]:
-            assert isinstance(expectation_value, float)
-            assert expectation_value > 0.0
-            assert expectation_value < 1.0
+        ]["real"][1:]:
```

Comment:
You're selecting a range here (`1:`) and then you ask whether it's `float`.
Shouldn't it be `[1]` ?

Reply:
No, we're for looping over the array and making sure each element is a float

Reply:
Oh, it's inside for!
Sorry, missed that!